raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -128,7 +128,7 @@ def ping_pong_message_success(transport0, transport1):
     transport1._raiden_service.sign(pong_message)
     transport0.send_async(queueid1, ping_message)
 
-    with Timeout(20, exception=False):
+    with Timeout(40, exception=False):
```

Comment:
why did you change from 20 to 40?

Reply:
I think it's a tradeoff. For the CI long timeouts mean it might take long until tests fail, locally tests might fail because the timeouts are not long enough. I had this today so I changed it to not have test failure because my machine is to slow.

Reply:
Can you make this a constant, so that we don't have a central location to change these timeouts.